Zend/Optimizer/zend_func_info.c

```diff
@@ -884,7 +884,7 @@ ZEND_API uint32_t zend_get_func_info(
 		}
 		if (!ret) {
 			ret = zend_get_return_info_from_signature_only(
-				callee_func, /* TODO: script */ NULL, ce, ce_is_instanceof);
+				callee_func, /* TODO: script */ NULL, ce, ce_is_instanceof, /* use_tentative_return_info */ 0);
```

Comment:
Same here, we should pass `!call_info->is_prototype` (though in this case it shouldn't really matter).